api/scanner/exif/exif_test.go

```diff
@@ -65,16 +66,61 @@ func TestExifParsers(t *testing.T) {
 		},
 		{
 			path: "./test_data/stripped.jpg",
-			assert: func(t *testing.T, exif *models.MediaEXIF) {
-				assert.Nil(t, exif)
+			assert: func(t *testing.T, exif *models.MediaEXIF, err error) {
+				assert.NoError(t, err)
+				if exif == nil {
```

Comment:
Is there any reason to change this test case? If the file doesn't have exif, this should be `nil`, but not an instance with zero value.

Reply:
the reason is that the internal and external parsers return different results, as I mentioned in my previous comment: in 1 case it is a simple `nil`, while in another - an object with empty values

Reply:
SG.